test.sh

```diff
@@ -32,7 +32,7 @@ TESTS=$(go list -f '{{ if .TestGoFiles }} {{.ImportPath}} {{end}}' ./...)
 go test -v ${TESTS}
 
 # Ignore these paths in the following tests.
-ignore="vendor\|\_gopath\|assets.go"
+ignore="vendor\|\_gopath\|\assets.go\|\constants_gendocs.go"
```

Comment:
Licence header. Copy from one of the other go files, run `make test` locally to confirm.